documentation/images/plantuml/boardfarm_prplMesh_classes.puml

```diff
@@ -96,7 +96,7 @@ package prplMesh_entities <<Frame>> {
         get_rxpower()
```

Comment:
All of these don't exist at the moment. get_mac() will never exist - mac is accessed directly. status I'm not convinced if we will add it. tx and rxpower we may add in the future. So I propose to remove status and get_mac(). Also, mark the other functions as abstract please.